terraform/bambictf.tf

```diff
@@ -196,6 +204,34 @@ systemctl start wg-quick@internal
 TERRAFORMEOF
 }
 
+resource "hcloud_server" "moloch" {
+  name        = "moloch${count.index + 1}"
+  image       = data.hcloud_image.bambimoloch.id
+  location    = "fsn1"

```

Comment:
Could you rebase/merge the master branch and use `local.location` here?

Reply:
should be fixed